src/main/interface/msp.c

```diff
@@ -1755,7 +1758,12 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             currentPidProfile->itermAcceleratorGain = sbufReadU16(src);
         }
         if (sbufBytesRemaining(src) >= 2) {
-            currentPidProfile->dtermSetpointWeight = sbufReadU16(src);
+            sbufReadU16(src); // was currentPidProfile->dtermSetpointWeight
```

Comment:
Shouldn't it just at all axes to keep the behavior?

Reply:
Old code was using this single uint16_t value for all axes effectively, what I say is let's use this value for setting all axes at once?

Reply:
It's not really a valid use case to override all 3 axes with a single value and I think it would actually be a bug causing unexpected results.  Likewise, what value would we write out through MSP if the user had set r/p/y differently in the CLI?  While it's true that roll/pitch will often have the same or similar values, yaw will tune differently.

Reply:
And before there was only R/P - there was no feedforward on yaw.

Reply:
What we _could_ do (and have done in other places) in order to avoid breaking clients that do not yet support the new format:
- read the value for `dtermSetpointWeight`;
- if the extra bytes at the end for the individual FF parameters aren't send, use the legacy `dtermSetpointWeight` to set the parameters (potentially scaling down yaw to make it reasonable), otherwise ignore it;
- return an average value of the FF parameters for `dtermSetpointWeight` when read over MSP.

Reply:
Seems like that would lead to some strange results ending up in the axes.  We don't have any idea yet whether people will prefer higher yaw feedforward as compared to roll/pitch so we don't really know to scale up or down for yaw. Also up until this point people haven't had a way to configure roll/pitch separately so it might become a standard practice to have pitch a little higher than roll now that they can.  Technically feedforward pitch needs to be a little higher to get the same response on most quads due to more inertia on the pitch axis.

I'd be more open to just doing it for roll/pitch but trying to mix in yaw would be questionable.

Reply:
Yaw feed forward requirement is very different from pitch and roll, at least that's been my experience in flight testing.  That's why I'd like to keep yaw independently adjustable.  I do see value in being able to independently adjust pitch and roll FF, since many freestyle quads have more inertia in pitch and would need more FF to retain a balanced flight characteristic.   In the case of a user wanting to not use FF and by only setting D weight and leaving FF at zero, I'd suggest they get FF on pitch and roll via D weight, however, assuming D on yaw is zero, they will get no FF on yaw.  If they were to set D on yaw to a significantly non-zero value the quad will become un-flyable from yaw oscillation, so in practice they will not reap any benefit from FF on yaw, and might thereby be persuaded to use the FF method over D weight.